server/lib/util.js

```diff
@@ -91,3 +108,28 @@ function take() {
     });
   };
 }
+
+/**
+ * String to boolean
+ * @function stringToBool
+ * @param {string} value The string to evaluate
+ * @return {boolean}
+ */
+function stringToBool(value) {
+  return (value + '').toLowerCase() === 'true';
+}
+
+/**
+ * nullString
+ * @function nullString
+ * @param {string} value The string to evaluate
+ * @return {boolean}
+ */
+function nullString(value) {
+  value += '';
+  return value.toLowerCase() === 'null' ||
```

Comment:
As a small increase in efficiency, you could do what you did above:
```js
// convert to lower case string
value = (value + '').toLowerCase();
```
Also, you could .reduce the amount of code you write.
```js
// falsey value mask
const falsey = ['null', 'undefined', 'false', '0', ''];
return falsey.contains(value);
```